modules/rostests/apitests/ntdll/CMakeLists.txt

```diff
@@ -62,7 +68,8 @@ if(ARCH STREQUAL "i386")
     add_asm_files(ntdll_apitest_asm i386/NtContinue.S)
 endif()
 
-add_executable(ntdll_apitest ${SOURCE} ${ntdll_apitest_asm} testlist.c)
+add_rc_deps(testdata.rc load_notifications)
```

Comment:
This uses OBJECT_DEPENDS, meaning it expects a file path, not a target.

Reply:
So `$<TARGET_FILE:load_notifications>` would be more appropriate, thanks.
(Ps i'll also fix this in `modules/rostests/apitests/apphelp/CMakeLists.txt`)

Reply:
The documentation doesn't mention that it supports generator expressions, so some experimentation may be needed.

Reply:
It does not seem to support it.

Reply:
Guess it's gonna be `${CMAKE_CURRENT_BINARY_DIR}/load_notifications/load_notifications.dll` .